src/frontend/pages/postings/forms/EditPostingsForm.js

```diff
@@ -317,7 +306,6 @@ const EditPostingsForm = () => {
       {reqNotFilled && (
         <RequiredText>Please fill out all required fields.</RequiredText>
       )}
-      {renderAddNewDialog()}
```

Comment:
Are you sure we don't need this anymore ? What was the purpose of this dialog ?

Reply:
I believe we don't need this anymore. This function was used to render a dialog, so for example, when you wanted to add a new requirement, you would press the "Add New+" button to generate the following dialog:
![image](https://user-images.githubusercontent.com/82276831/190532242-0bfd9564-1fa6-44cc-8f36-b838240d13a1.png)
However, now that we using multiline inputs for each of these fields, there's no more use for a dialog and thus this function as well. I also deleted all the other dialog functions